Library/Formula/ffmpeg.rb

```diff
@@ -3,7 +3,7 @@ class Ffmpeg < Formula
   url "https://ffmpeg.org/releases/ffmpeg-2.6.1.tar.bz2"
   sha256 "a4f6388706ee2daba9d35d2aa018ae5feeb450efa716555e011a6543d43ec7c1"
 
-  head "https://git.videolan.org/ffmpeg.git"
+  head "git://source.ffmpeg.org/ffmpeg.git"
```

Comment:
if the [github mirror](https://github.com/FFmpeg/FFmpeg) is always up to date then would it be worth using it instead for the benefits of `https://` over `git://`?